Add `SVS` and `SVS2` contest articles (#7450)
ppy/osu-wiki: wiki/Contests/SVS/1/en.md

---
tags:
  - svs
  - svs1
  - svs 1
  - svs#1
---

# Slider Velocity Showdown

![SVS logo](/wiki/contests/svs/img/Banner.png)

> The **Slider Velocity Showdown** (***SVS***) was the 1st iteration of the osu!taiko mapping contest with a focus on creativity and innovation. This contest was open to all that want to flex their creative muscles, whether they’ve never placed an inherited point in your life or if they’ve been mapping for years.

Reviewer: ```suggestion
The **Slider Velocity Showdown** (***SVS***) was the second iteration of the osu!taiko mapping contest with a focus on creativity and innovation. This contest was open to anyone who wanted to flex their creative muscles, whether they have never placed an inherited point in their life or if they have been mapping for years.
```

## Contest schedule

| Event | Timestamp (UTC) |
| --: | :-- |
| Registration phase | 2021-07-01/2021-08-01 23:59 |
| Submission phase | 2021-08-01/2021-09-01 23:59 |
| Judging phase | 2021-09-01/2021-09-14 |

> | Results announcement | 2021-09-18 |

Reviewer: the phases overlap according to the timestamp, is that intentional?

Author: adjusted

| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1361851)
- [Discord server](https://discord.gg/8WbX3XdmTs)
- [Results sheet](https://docs.google.com/spreadsheets/d/1ketuj4BgsevTpKVIM16d7drEdLYYYntPNdX2uIHfpk4/edit?usp=sharing)

## Podium

This competition has come to an end and resulted in the following podium:

| Placing | User |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_NL] [Cookie_Tree](https://osu.ppy.sh/users/502722) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_JP] [Grape_Tea](https://osu.ppy.sh/users/9540073) |

## Ruleset

### Contest rules

- **The contest will consist of one round where contestants will map any song of their choosing.** The chosen song must not breach the osu! [song content rules](/wiki/Rules/Song_Content_Rules); if you wouldn’t upload it to your profile, then don’t use it here. Anything that violates the osu! rules will be disqualified.

> - **Each contestant is limited to a single difficulty of any level.**

Reviewer: level?

Author: level as in difficulty level (kantan, futsuu, muzukashii, oni, inner oni), host probably worded it that way to avoid repeating the word "difficulty"

Reviewer: then it'd be something like

```suggestion
- **Each contestant is limited to one of each difficulty level.**
```

Author: wouldn't that imply they have to map 5 diffs? one of each difficulty level? the contest takes only 1 diff, and it can be of any difficulty level.

Reviewer: uh ok then i guess we'll just keep this wording

- **Your map must be an osu!taiko map.**

> - **Every entry should utilize only timing points (BPM changes) and inherited points (slider velocities).** NThe maps don’t need to be rankable; in fact, It's to give up on any chance of making a rankable map. If it helps, think of this like Aspire, but incredibly simplified. All entries that include storyboarding, hitsounding, and skinning will be disqualified. You may use sliders and spinners. Fringe entries will be dealt with on a case-by-case basis.

Reviewer: ```suggestion
- **Every entry should utilise only timing points (BPM changes) and inherited points (slider velocities).** The maps do not need to be rankable; in fact, it is to give up on any chance of making a rankable map. If it helps, think of this like Aspire, but incredibly simplified. All entries that include storyboarding, hitsounding, and skinning will be disqualified. You may use sliders and spinners. Fringe entries will be dealt with on a case-by-case basis.
```

- **Maps must not be uploaded before results are announced.** This includes maps that were created with no intention of being entered in this contest. Anybody who shares their map in any way before the announcement post is up will be disqualified from the contest.

> - **Submissions must be submitted via .osz format.** Sending in `.osu` files only is not allowed. If you don’t know how to export a .osz file, please follow [this guide](/wiki/Client/File_formats). All `.osu` files submitted will be disqualified if a .osz is not sent afterwards.

Reviewer: ```suggestion
- **Submissions must be submitted via `.osz` format.** Sending in `.osu` files only is not allowed. If you don’t know how to export a `.osz` file, please follow [this guide](/wiki/Client/File_formats). All `.osu` files submitted will be disqualified if a `.osz` file is not sent afterwards.
```

- **Submissions will not be accepted after the submission deadline.** No exceptions. Only the most recent entry will be considered.
- **In order to participate, you must join the [Discord server](https://discord.gg/8WbX3XdmTs) for the contest and register as a participant.** This is purely for administrative purposes in order to keep track of who is who and make the contest run smoothly.

> - **Songs must be under 5 minutes in length.** Judging requires maps to be played multiple times, this rule is just to be respectful of their time.

Reviewer: ```suggestion
- **Songs must be under 5 minutes in length.** Judging requires maps to be played multiple times. This is in order to respect the time of the judges.
```

### Judging criteria

> Judging will be based on a holistic and generic scoring method. Since creativity is incredibly subjective and the placements of this contest are in no way indicative of objective quality, this contest has three loosely-defined categories that contestants should aim to fulfill:

Reviewer: ```suggestion
Judging will be based on a holistic and generic scoring method. Since creativity is incredibly subjective and the placements of this contest are in no way indicative of objective quality, this contest has three loosely defined categories that contestants should aim to fulfill:
```
